src/test/resources/Features/OCD-1776-SEDTweaks.feature

```diff
@@ -1,33 +1,31 @@
 @Regression
-Feature: SED Display in product details
+Feature: OCD-1776 - SED Display in product details
   This test is to verify text changes and other display updates to SED section
 
-  Scenario: Verify SED details
-    Given I am on the Details page of Listing "8338"
+  Scenario Outline: Verify SED details
+    Given I am on listing details page of listing with database ID "<DB_ID>"
     Then I should see these accordions exist on the details page: Criteria, CQMs, SED, G1/G2, Surveillance, Additional Info
     And I should see the accordions in the following order: Criteria, CQMs, SED, G1/G2, Surveillance, Additional Info
+    Examples:
+		|DB_ID|
+		|8338|
 
-  Scenario: Verify usability report title update
-    Given I am on the Details page of Listing "8338"
-    When I open SED details accordion
-    Then usability report text should read as 'Full Usability Report'
-
-  Scenario: Verify SED Intended User Description header
-    Given I am on the Details page of Listing "8338"
-    When I open SED details accordion
-    Then intended user description header should show as 'Description of Intended Users'
-
-  Scenario: Verify SED download option and button title
-    Given I am on the Details page of Listing "8338"
-    When I open SED details accordion
-    Then the download button title should read as 'Download SED Details'
+  Scenario Outline: Verify usability report title update
```

Comment:
Why is this an Outline too? It's not checking differences between Listings, right?